### DISCUSSION TITLE
PICKCHARS, displaying multiple times before auto-type execution?

### DISCUSSION THREAD

**Discussion by r3times** [keepassxreboot/keepassxc#9130] [Q&A] [answered]
I was delighted to see the extension of the PICKCHARS function to attributes other than the password alone. Is there a Keepassxc equivalent for displaying the PICKCHARS dialog box any number of times and subsequently performing the auto-type?

https://keepass.info/help/base/placeholders.html#pickchars

I have to use this frequently for multiple banks, successfully so in Keepass, but when trying to invoke the following example in Keepassxc, a syntax error message is displayed:

`{PICKCHARS:Memorable:ID=1, C=1}{TAB}{PICKCHARS:Memorable:ID=2, C=1}{TAB}{PICKCHARS:Memorable:ID=3, C=1}`

A similar question asked last year, a little lost among a series of queries: #https://github.com/keepassxreboot/keepassxc/discussions/7349#discussioncomment-2067339

My eyes are desperate to move on from trying to use Keepass under Mono, hoping it's something I've overlooked in the instructions!

Many thanks.

**Comment by droidmonkey** [comment, marked as answer]
We don't support that, you are being a syntax error because of the extra parameters (the second :...)

You don't need to do this anyway because we support pressing tab between pickchars typing. It's a checkbox in the dialog.

**Reply by r3times** [reply]
> ... we support pressing tab between pickchars typing. It's a checkbox in the dialog.

Brilliant...it just works, and that addition of the checkbox makes it simple to account for banks and their random tab choices.

For others, my PICKCHARS entry, imported from Keepass and shown above, becomes `{PICKCHARS:Memorable}`, I choose the three numbers requested by my bank, then check or uncheck the "press tab between characters" box as required. Much more elegant.

Already looking forward to practicing, thanks so much!
